core/testMinimumPhpVersion.php

```diff
@@ -22,14 +22,21 @@
 // 3) composer.json (in two places)
 // 4) tests/PHPUnit/Integration/ReleaseCheckListTest.php
 $piwik_minimumPHPVersion = '7.2.5';
+$piwik_maximumPHPVersion = '8.0.0';
 $piwik_currentPHPVersion = PHP_VERSION;
 $minimumPhpInvalid = version_compare($piwik_minimumPHPVersion, $piwik_currentPHPVersion) > 0;
+$maximumPHPInvalid = version_compare($piwik_maximumPHPVersion, $piwik_currentPHPVersion) >= 0;
```

Comment:
Wait... no 
`$maximumPHPInvalid = version_compare($piwik_maximumPHPVersion, $piwik_currentPHPVersion) <= 0;`

Reply:
I think that syntax is really confusing, so I'd go with the alternative one: (I hope I didn't mix it up)
https://www.php.net/manual/en/function.version-compare.php
```php
$maximumPHPInvalid = version_compare($piwik_maximumPHPVersion, $piwik_currentPHPVersion, '>=');

Reply:
I think it should be 
`$maxVersionInvalid = version_compare($maxVersion, PHP_VERSION, '<=');` as we are checking if the version is below 8.0.0 rather than above. Otherwise this option should clear things up, yes.

Reply:
You are of course right.